webapp/src/components/common/Matrix/components/MatrixUpload/index.tsx

```diff
@@ -0,0 +1,80 @@
+/**
+ * Copyright (c) 2024, RTE (https://www.rte-france.com)
+ *
```

Comment:
Only a index.tsx => MatrixUpload.tsx directly

Reply:
I was planning to add tests, but true